cmd/gossamer/configcmd.go

```diff
@@ -56,7 +57,7 @@ var (
 )
 
 // makeNode sets up node; opening badgerDB instance and returning the Dot container
-func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
+func makeNode(ctx *cli.Context, gen *genesis.GenesisState) (*dot.Dot, *cfg.Config, error) {
```

Comment:
We don't do anything with it yet? Should we be adding the bootnodes to the list of nodes?

Reply:
yeah, we are going to be using it in the future. going to save it for my next PR